fe2-android/app/src/main/java/com/github/dedis/student20_pop/detail/LaoDetailViewModel.java

```diff
@@ -219,19 +267,22 @@ public void sendVote(List<ElectionVote> votes) {
                                     answer -> {
                                         if (answer instanceof Result) {
                                             Log.d(TAG, "sent a vote successfully");
+                                            // Toast ? + send back to election screen or details screen ?
```

Comment:
Is this comment an implementation to be done? You can change this to a TODO

Reply:
Indeed, I forgot about that line. I'll just change it to a Toast